packages/ng-query/src/lib/is-mutating.ts

```diff
@@ -24,17 +24,20 @@ class IsMutating {
   ): Observable<number> {
     const [filters] = parseMutationFilterArgs(arg1, arg2);
 
-    return new Observable<number>((obs) =>
+    return new Observable<number>((obs) => {
+      obs.next(0);
```

Comment:
saw this in `vue-query` tests, shouldn't initial value be 0 ? @NetanelBasal

Reply:
Yep

Reply:
Hmmm, actually, not. Where did you see this?

Reply:
https://github.com/TanStack/query/blob/main/packages/vue-query/src/__tests__/useIsMutating.test.ts#L16

Reply:
We should take the observer.getCurrentResult() and not hard-coded 0

Reply:
I believe it should be `obs.next(this.instance.isMutating(filters))` in this case.